crates/viewer/re_space_view_map/src/visualizers/mod.rs

```diff
@@ -73,6 +75,28 @@ impl GeoSpan {
     }
 }
 
+/// A picked instance.
+#[derive(Debug, Clone)]
+pub struct PickedInstance {
+    pub instance_path: InstancePath,
+
+    /// Keep track of the mouse distance from the object's center, so we can arbitrate.
+    pixel_distance: f32,
```

Comment:
This is a bit confusing, since `egui` uses "pixel" to mean "physical pixels", and otherwise uses the term "ui points" as its distance metric